config/settings.yml

```diff
@@ -113,6 +113,7 @@ evss:
     region: evss_s3_region
   disability_compensation_form:
     timeout: 55
+    submit_timeout: 355
```

Comment:
Someone once recommended to me that for any of these types of numbers, perhaps add a comment next to them informing future devs what this value might translate to.  It has helped me when I've gone back to look at the code.

For example:

```ruby
submit_timeout: 355 # 6 minutes
```

Reply:
I wash just talking about the importance of comments... so I agree with this :)